### PR TITLE
Global: Set up new repository structure and set up Yarn Workspaces

### PR DESCRIPTION
**Pull request by Fallion**
- Moves Kiwi Frontend files into /kiwicom
- Sets up Global ESLint rules
- Sets up Global Flow
- Sets up Global Babel

### CODE REVIEW

**Comment by michalsanger** [comment]
Naming suggestion - rename `frontend` to `kiwi.com`

File: package.json [hunk 1]
```diff
@@ -46,20 +19,8 @@
...
-    "setupFiles": [
-      "raf/polyfill",
-      "./etc/enzymeSetup"
-    ],
-    "snapshotSerializers": [
-      "enzyme-to-json/serializer"
-    ]
-  }
+  "private": true,
+  "workspaces": ["blkn", "frontend"]
```
Comment (oreqizer) [inline package.json]: `frontend` -> `kiwi.com`

File: kiwi.com/package.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    "raf": "^3.4.0",
+    "react-test-renderer": "^16.2.0",
+    "rimraf": "^2.6.2"
+  },
+  "jest": {
+    "setupFiles": ["raf/polyfill", "./etc/enzymeSetup"],
+    "snapshotSerializers": ["enzyme-to-json/serializer"]
+  },
+  "private": true,
+  "workspaces": ["blkn", "frontend"]
```
Comment (oreqizer) [inline kiwi.com/package.json]: `frontend` -> `kiwi.com`

**Comment by oreqizer** [comment]
dunno about dots, how about `kiwicom`?

**Pull request merged**